src/container/Page/Page.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react'
+import React, { useContext } from 'react'
```

Comment:
Does this need to be here?

Since I don't see it used.

Reply:
Oh good point thx